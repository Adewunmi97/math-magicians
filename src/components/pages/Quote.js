import React from 'react';
import '../../App.css';
import './Welcome.css';

function Quote() {
  return (
    <div className="welcome-container">
      <div className="info">
        <h1> Quote</h1>
        <p>
          We will always have STEM with us. Some things will drop out of the public eye
          and go away,but there will always be science, engineering, and technology.
          And there will always, always be mathematics.
          — Katherine Johnson, African-American mathematician
        </p>

        <p>
          {' '}
          Mathematics as an expression of the human mind reflects the active will,
          the contemplative reason, and the desire for aesthetic perfection. Its basic
          elements are logic and intuition, analysis and construction, generality
          and individuality.
          — Richard Courant, German-American mathematician
        </p>
      </div>
    </div>
  );
}

export default Quote;
