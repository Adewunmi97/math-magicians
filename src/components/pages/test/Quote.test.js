import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Quote from '../Quote';

describe('test Quote component functionality', () => {
  it('should render Quote heading correctly', () => {
    const { container, getByRole } = render(<Quote />);
    expect(getByRole('heading')).toHaveTextContent('Quote');
    expect(container).toMatchSnapshot();
  });
});
