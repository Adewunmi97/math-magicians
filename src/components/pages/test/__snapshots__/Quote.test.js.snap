// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Quote component functionality should render Quote heading correctly 1`] = `
<div>
  <div
    class="welcome-container"
  >
    <div
      class="info"
    >
      <h1>
         Quote
      </h1>
      <p>
        We will always have STEM with us. Some things will drop out of the public eye and go away,but there will always be science, engineering, and technology. And there will always, always be mathematics. — Katherine Johnson, African-American mathematician
      </p>
      <p>
         
        Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality. — Richard Courant, German-American mathematician
      </p>
    </div>
  </div>
</div>
`;
